project_reloader_plugin.py

```diff
@@ -10,46 +10,58 @@
 
 class ProjectReloaderPlugin(QgsServerFilter):
     """
-    This plugin checks if the project is loaded via postgresql.
-    it then check periodically if the project needs to be reloaded.
+    This plugin checks if the project from the Map Request value is loaded.
+    it then check periodically if current requested project needs to be reloaded.
```

Comment:
This plugin checks if the project from the Map Request value is loaded => does not make sens anymore.
This should be enought:

On each request this plugin check the project modification time and reload the project if needed.